api/src/users/users.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable, Inject } from "@nestjs/common";
+import { Repository } from "typeorm/repository/Repository";
+import { User } from "./entities/user.entity";
+import { CreateUserDto } from "./dto/create-user.dto";
+import { USER_REPOSITORY } from "src/constants";
+
+@Injectable()
+export class UsersService {
```

Comment:
suggestion here: you can actually create a generic `repo` which you called `service` and extend from it with the Type inside the <>, like <User>, just to enforce the usage of at least standard methods, like `find`, `findAll`, `findOne`, `create` and `remove`

Reply:
This repository must be changed to a simpler structure.